### ISSUE TITLE
Percolator "Picked protein FDR (-f)" option requires flanking amino acids in PIN peptide column

### ISSUE THREAD

**Issue by tivdnbos** [compomics/ms2rescore#24]
see more at: https://github.com/percolator/percolator/wiki/Protein-inference-example

**Comment by RalfG** [comment]
I think you should already be able to do this with the `percolator` option in the configuration file: 
```json
{
    "$schema": "https://raw.githubusercontent.com/compomics/ms2rescore/master/ms2rescore/package_data/config_schema.json",
    "percolator": {
        "f": "path/to/sequence_db.fasta"
    }
}
```
or 

```json
{
    "$schema": "https://raw.githubusercontent.com/compomics/ms2rescore/master/ms2rescore/package_data/config_schema.json",
    "percolator": {
        "picked-protein": "path/to/sequence_db.fasta"
    }
}
```

See https://github.com/compomics/ms2rescore/blob/master/configuration.md.

Let me know if it works!

**Comment by tivdnbos** [comment]
It creates the following error message for the first PSM listed.

Exception caught: ERROR: Reading tab file, the peptide sequence <sequence> with PSM id <PSM id> does not contain one or two of its flanking amino acids.

**Comment by tivdnbos** [comment]
FYI: If "tandem2pin" is used (not the one in rescore), the flanking amino acids do appear in the Peptide column ([flanking AA].[Peptide].[flanking AA]).

**Comment by RalfG** [comment]
Implemented for X!Tandem in #28

**Comment by Hassan-1991** [comment]
Hi, has a solution for this been implemented in the MSGF+ms2rescore pipeline? I'm getting the exact error mentioned by tivdnbos above.
